packages/shopify-codemod/test/fixtures/conditional-assign-to-if-statement/assign-to-other-member.output.js

```diff
@@ -0,0 +1,3 @@
+this.foo || (this.bar = 'lol');
+self.foo || (self.bar = 'lols');
+x || (y = 'lols');
```

Comment:
I'm wondering if we should just transform all of these, too — we don't really like this style, regardless of whether it's being assigned to the same thing or not.